build_tools/cmake/iree_benchmark_suite.cmake

```diff
@@ -237,10 +237,51 @@ function(iree_benchmark_suite)
         add_dependencies(iree-benchmark-suites "${_TRANSLATION_TARGET_NAME}")
       endif(NOT TARGET "${_TRANSLATION_TARGET_NAME}")
 
+      set(_COMPILATION_STATS_TRANSLATION_TARGET_NAME
+        "${_TRANSLATION_TARGET_NAME}-compilation-stats"
+      )
+      set(_COMPILATION_STATS_VMFB_FILE
+        "${_VMFB_ARTIFACTS_DIR}/${_MODULE_SOURCE_BASENAME_WITH_HASH}-compilation-stats.vmfb"
+      )
+      if(IREE_ENABLE_COMPILATION_BENCHMARKS AND NOT TARGET "${_COMPILATION_STATS_TRANSLATION_TARGET_NAME}")
+        iree_bytecode_module(
+          NAME
+            "${_COMPILATION_STATS_TRANSLATION_TARGET_NAME}"
+          MODULE_FILE_NAME
+            "${_COMPILATION_STATS_VMFB_FILE}"
+          SRC
+            "${_MODULE_SOURCE}"
+          FLAGS
+            "--iree-vm-emit-polyglot-zip=true"
+            "--iree-llvm-debug-symbols=false"
+            ${_TRANSLATION_ARGS}
```

Comment:
Can you add some comments for each flag, indicating why they are specified just for the "compilation benchmarks"? (I think I follow by reading through the linked issues, but the high level summary should be recorded here)

Reply:
Done.